packages/adminrouter/extra/src/README.md

```diff
@@ -479,6 +479,13 @@ as `.js` and `.css` files. This is due to the fact that the linked resources wil
 be reachable only in their relative location
 `<dcos-cluster>/services/<application ID><link>`.
 
+### Disabling URL path rewriting for selected applications
+Some applications like Jenkins or Artifactory, permit specifying a context path (e.g. `/service/jenkins`) that is used while generating application URLs (both relative and absolute). However, normally Adminrouter strips these, and this can lead to breakages, esp. for assets like JavaScript files and images.
+
+It is possible to workaround it by using another Nginx in the application container which rewrites URLs from / to /service/jenkins. This isn't ideal though, so an extra Marathon label is recognized by Admin Router. If a task has `DCOS_SERVICE_REWRITE_REQUEST_URLS` label set to `no` or `0` or `false`, Admin Router will not strip the context path and the upstream request will be made with the same the URL path as the client request has been made.
+
```

Comment:
In general I think this documentation should present a Marathon application definition snippet that shows how to properly set this in practice (as done above).

> `no` or `0` or `false`

In general, it's preferable to introduce _one_ definite way (here we offer three) and docs that are precise about that way (`false` as written here could be both, a JSON string or a JSON boolean)

I understand that we should also should "be liberal in what you expect" to a certain degree -- what do you think about only supporting both, the JSON string `"false'` and the JSON boolean `false` (and to document it that way)?